tooling/cli/Cargo.toml

```diff
@@ -6,7 +6,7 @@ name = "tauri-cli"
 version = "1.5.9"
 authors = [ "Tauri Programme within The Commons Conservancy" ]
 edition = "2021"
-rust-version = "1.60"
```

Comment:
Even though this is a CLI and a MSRV shouldn't matter that much but I think we plan to keep 1.60 for 1.x. Anyways, this PR probably needs to target `dev` not `1.x` so this shouldn't be a problem.

Reply:
removed!